Style/index.css

```diff
@@ -1,5 +1,52 @@
-#divBody{
-	padding-left: 20%;
-	padding-right: 20%;
-	padding-top: 2%;
+* {
+	margin: 0;
+	padding: 0;
+}
+
+h1 {
+	padding: 1em;
+}
+
+h1 img {
+	width: 100%;
+	margin: auto;
+	display: block;
+	max-width: 800px;
+}
+
+menu {
```

Comment:
It's generally better practice (and better for the network!) if you try to have just a single CSS file, instead of a gazillion. I moved banner.css into index.css.